.pylintrc

```diff
@@ -531,5 +531,5 @@ min-public-methods=2
 
 # Exceptions that will emit a warning when being caught. Defaults to
 # "BaseException, Exception".
-overgeneral-exceptions=BaseException,
-                       Exception
+overgeneral-exceptions=builtins.BaseException,
+                       builtins.Exception
```

Comment:
Could you please point where I can learn the difference between builtins and "non-builtins" exceptions?
I tried to find it but was not trivial to get a useful link...of course it does not seem really relevant if you don't have anything in particular to add on that.

Reply:
Ah, this I forgot to mention in the log. But given a new enough pylint and python, it will tell you that not specifying "builtins." as here is deprecated.

Reply:
Basically they will stop supporting if `builtins` is not used. Interesting.